tests/integration/targets/prepare_jinja2_compat/filter_plugins/jinja_compatibility.py

```diff
@@ -1,6 +1,11 @@
-# Copyright (c) Ansible Project
-# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
-# SPDX-License-Identifier: GPL-3.0-or-later
+# This code is part of Ansible, but is an independent component.
```

Comment:
Also, `part of Ansible` is not entirely actuate.

Reply:
This is what we use in many community collections. I think this PR is the wrong time and place to change that formulation.